src/Enghouse/readme.md

```diff
@@ -0,0 +1,100 @@
+
+The communication uses the EI CustomChat widget component.  This can be referenced using the EI **CCaaS** default page or the [hosting page](./client/indexWatson.html) can set the `watsonAssistantChatOptions` prior to loading the EnghouseCCaaSWatsonAssistant.js for more advanced features, e.g.:
+
+```
+<script>
+  window.watsonAssistantChatOptions = {
+	// ... add any additional configuration options you may need..
+	integrationID: [integrationID],
+	region: [region],
+	serviceInstanceID: [serviceInstanceID],
+	subscriptionID: [subscriptionID],
+    carbonTheme: 'g100',
+	
+	serviceDeskFactory: (params) =>
+	  new window.EnghouseCCaaSWatsonAssistantSD(params.callback, ccName, ccUrl, agentIcon, webChatInstance),
+
+	onLoad: async (instance) => {
+	  webChatInstance = instance;
+	  instance.render();
+	}
+  };
+
+</script>	
+<script src="https://ccsp.ei.eilabonline.biz/PS.CustomChat/EnghouseCCaaSWatsonAssistant.js"></script>
+```
+
+
+## Setting Up
+
+
+The integration utilises the flexible Custom Chat integration to EI **CCaaS**, the steps for deployment are:
+
+
+1. Configure Chat routing in EI **CCaaS** tenant with desired Queue and Skills.
+2. Configure CustomChat "widget" and copy its URL used to control the Live Agent interaction specific to your needs, and a "WA" URL to access the WA integration from EI **CCaaS**.
+3. When using the CCaaS integration HTML, the "widget" URL is required to be passed either as the *ccURL* parameter to the "WA" URL, or as a variable in the WA Dialog called *CCURL*.  The "WA" URL must also include parameters to utilise your Assistant's *integrationID*, *region*, and *serviceInstanceID*.  If using your own HTML you can hard-code the required parameters, or obtain the values via some alternative method to suit your preference.
+4. By default the name presented to the contact-centre is the Watson user_id - this display name can be changed in the Flow by setting variables (case of names ignored) as per [Dialog example](./VG_TravelDemo-dialog.json), i.e. 
+- firstName - if this is set without lastName, this is the display name
+- lastName - if the firstName is also set, the display name is firstName and lastName (with a space between)
+- userId - if the firstName is not set this may set the display name
+
+
+## Example
+
+If the CustomChat "widget" URL was:
+
+`https://{Enghouse CCaaS FQDN}/SocialConnectorHelperAPI/api/customchat/getWidgetConfiguration?WidgetId=<*widgetID*>`
+
+...you would set this to the variable *CCURL* in the Dialog, prior to any likelihood of needing to speak to a Live Agent.
+
+
+If the Assistant URL provided to you by Enghouse Interactive was:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat`
+
+...you would access your Assistant dialog pass your Assistants' parameters to it (using default EI **CCaaS** Watson page in this example):
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>`
```

Comment:
Since I don't believe customers are going to want to actually you use your `PS.CustomChat/indexWatson.html` and will use their own page instead, I would just remove the references to it from this documentation and the entire `URL options` section below. I think customers are likely to get confused as to what these values are and where they would use them.

If you really want to keep this information, then I would put it in a separate section by itself and make it clear that this only needs to be used if the customer does not need to use their own HTML file and just wants an externally hosted web chat page.